utils/demarchesSimplifiees.js

```diff
@@ -51,83 +48,57 @@ function getUuidDossierNumber(number) {
+
+  psy.address = getChampValue(dossier.champs, 'Adresse du cabinet');
+  psy.departement = getChampValue(dossier.champs, 'Votre département'); // "14 - Calvados"
+  psy.phone = getChampValue(dossier.champs, 'Téléphone du secrétariat');
+  psy.teleconsultation = parseTeleconsultation(
     getChampValue(dossier.champs, 'Proposez-vous de la téléconsultation ?'),
   );
-  const website = getChampValue(dossier.champs, 'Site web professionnel (optionnel)');
-  const email = getChampValue(dossier.champs, 'Email de contact');
-  const description = getChampValue(dossier.champs, 'Paragraphe de présentation (optionnel)');
+  psy.website = getChampValue(dossier.champs, 'Site web professionnel (optionnel)');
+  psy.email = getChampValue(dossier.champs, 'Email de contact');
+  psy.description = getChampValue(dossier.champs, 'Paragraphe de présentation (optionnel)');
 
   // @TODO comma separated values not reliable
-  const training = parseTraining(
+  psy.training = parseTraining(
     getChampValue(dossier.champs, 'Formations et expériences'),
   );
-  const adeli = getChampValue(dossier.champs, 'Numéro Adeli');
-  const diploma = getChampValue(dossier.champs, 'Intitulé ou spécialité de votre master de psychologie');
-  const languages = getChampValue(dossier.champs, 'Langues parlées (optionnel)');
-
-  const psy = {
-    dossierNumber,
-    state,
-    archived,
-    lastName,
-    firstNames,
-    address,
-    region,
-    departement,
-    phone,
-    website,
-    email,
-    personalEmail,
-    teleconsultation,
-    description,
-    training,
-    adeli,
-    diploma,
-    languages,
-  };
+  psy.adeli = getChampValue(dossier.champs, 'Numéro Adeli');
+  psy.diploma = getChampValue(dossier.champs, 'Intitulé ou spécialité de votre master de psychologie');
+  psy.languages = getChampValue(dossier.champs, 'Langues parlées (optionnel)');
 
   return psy;
 }
 
-function parsePsychologist(apiResponse) {
-  const dossiers = apiResponse.demarche.dossiers.nodes;
+function parsePsychologist(dossiers) {
   console.log(`Parsing ${dossiers.length} psychologists from DS API`);
 
-  if (dossiers.length > 0) {
-    const psychologists = dossiers.map((dossier) => parseDossierMetadata(dossier));
-    return psychologists;
-  }
-
-  return [];
+  return dossiers.map((dossier) => parseDossierMetadata(dossier));
 }
 
 /**
  * helper function called by getPsychologistList
  * @param {*} cursor
  * @param {*} accumulator
  */
-async function getAllPsychologistList(cursor, accumulator = []) {
-  const apiResponse = await graphql.requestPsychologist(cursor);
+async function getAllPsychologistList(graphqlFunction, cursor, accumulator = []) {
+  const apiResponse = await graphqlFunction(cursor);
 
-  const { pageInfo } = apiResponse.demarche.dossiers;
+  const { pageInfo, nodes } = apiResponse.demarche.dossiers;
 
-  const nextAccumulator = accumulator.concat(
-    parsePsychologist(apiResponse),
-  );
+  const nextAccumulator = accumulator.concat(nodes);
```

Comment:
No need to call parsePhychologist anymore here?
```suggestion
  const nextAccumulator = accumulator.concat(parsePsychologist(nodes));
```

Reply:
no, because this only get the data from graphql and concat them, if you want to do it it's after (in demarchessimplifiees.js: `
  const list = await getAllPsychologistList(graphql.requestPsychologist, cursor);
  list.psychologists = list.psychologists.map((psychologist) => parsePsychologist(psychologist));`